include/deal.II/fe/fe_nothing.h

```diff
@@ -277,7 +283,7 @@ class FE_Nothing : public FiniteElement<dim, spacedim>
 private:
   /**
    * If true, this element will dominate any other apart from itself in
-   * compare_for_face_domination();
+   * compare_for_domination();
```

Comment:
```suggestion
   * compare_for_domination(). This is because a space that only contains the zero function is definitely smaller
   * (and consequently dominant) when compared to any other finite element space.
```